README.md

```diff
@@ -1 +1,5 @@
 # php-lint-workflow-guide
+
+This repository serves as an example PHP project with the CI setup done for performing linting and code analysis as part of PHP coding standards check.
```

Comment:
Suggest that you add the php version and anything else which is relevant.

Reply:
Additional details about the project, including PHP version, have been added in the latest commit.